hubgrep/frontend_blueprint/templates/components/result_card/_result_card.scss

```diff
@@ -93,10 +93,4 @@
       display: none;
     }
   }
-}
-
-body[data-debug="true"] {
-  .item-detail-score {
-    display: unset;
-  }
```

Comment:
changed how we hide "debug" content by labeling stuff with debug class
if it has that class, hide it unless body has the debug attribute set to true